app/home/homeController.js

```diff
@@ -8,7 +8,9 @@
 
     function controllerFunction (cityService) {
           var vm = this;
-          vm.city = cityService.city;
+          vm.cityService = cityService;
+          // Curious as to why you need the whole object. Commented out in forecastController.
```

Comment:
Try removing the `ng-controller` attribute. I'm curious if that's an issue. Since you're already declaring `homeController` via `$stateProvider`.